source/ssb-chart/values.yaml

```diff
@@ -232,6 +232,98 @@ networkpolicy:
   # -- Optionally create your own NetworkPolicy for the current application.
   overrideDefaultPolicy: {}
 
+
+  # Two unique ID's are available for use with user/app authentication in todays platform
+  #
+  # 1. Keycloak unique client ID in the SSB Realm
+  #    This is "audience" referenced in the current IstioEndUserAuth config.
+  #    Caller and callee must agree on this authentication through the client config
+  #    and the validation.
+  #
+  # 2. Istio Spiffe ID/K8s SA
+  #   This is the "principal" referenced in AuthorizationPolicy manifest.
+  #
+  #   Applications must be referenced by their principal name
+  #   which is the unique Service Account name tied to a Spiffe/ID assigned by Istio.
+  #   This is needed to ensure the originity of the request (non-repudiation).
+  #
+  # The next (first) layer of security is Network policies. But note that these does not guarantee
+  # non-repuditation. Inside K8s applications in network policies are referenced by namespace and/or pod
+  # labels/names. These labels might not be unique.
+
+  # Policy example for an app with SA/name "app1" in namespace "n1"
+access:
```

Comment:
I'm a little unsure of how detailed I should go in the review.

Anyways, my comment here is that the default values here will render the example values in the output. 

I assume you still want to keep this for testing purposes, also after a merge with v4?

Reply:
I use this field to keep track of what i have implemented (ish.). In the next PRs i will uncomment blocks of this field, so in v4-branch this object will contain the full structure